src/rooms.py

```diff
@@ -0,0 +1,29 @@
+from room import Room
+from item import Item
+
```

Comment:
I love how you broke this into its own file, be careful with naming, a room and rooms file might be confusing at first glance for a future dev, maybe name it something like createworld...